### PR TITLE
[stable/fluent-bit] Add Pod Security Policy

### PR DESCRIPTION
**Pull request by gypsydiver**
Signed-off-by: Fernando Mendoza <fermendozarcs@gmail.com>

#### What this PR does / why we need it:
Add the option to create a Pod Security Policy.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

**Comment by gypsydiver** [comment]
@kfox1111 
@edsiper 
@hectorj2f

File: stable/fluent-bit/templates/psp.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
+{{- if .Values.rbac.pspEnabled }}
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: {{ template "fluent-bit.fullname" . }}
```
Comment (Flydiverny) [inline stable/fluent-bit/templates/psp.yaml:5]: Shouldnt this also have
```yaml
  labels:
    app: {{ template "fluent-bit.name" . }}
    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
    heritage: {{ .Release.Service }}
    release: {{ .Release.Name }}
```
 ? :)

**Pull request merged**